clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,74 @@
+# $testDuration controls
+# for how long the test should be executed
+# expressed in minutes
+{{$testDuration := 5}}
```

Comment:
Nit: personally I would suggest testDuration to be expressed in time.Duration-parsable format like "5m", "10s". Then it will be self explanatory and comments like "expressed in minutes" won't be necessary

Reply:
np, done.